kymatio/scattering2d/frontend/torch_f.py

```diff
@@ -0,0 +1,178 @@
+__all__ = ['Scattering2D']
+
+import torch
+from ..backend import cdgmm, Modulus, SubsampleFourier, fft, Pad, unpad, NAME
+from ..filter_bank import filter_bank
+from ..utils import compute_padding
+from ..scattering2d import scattering2d
```

Comment:
We should probably change the name of this function to distinguish it from the class `Scattering2D`. It's easy to confuse the two right now.

Another thing to think about is the directory structure. It may be better for the `scattering2d` function to be in a `core` submodule (or something like that). Right now, it's a little curious that the main user interface resides in a submodule, while the internal function implementing the transform resides in the module itself.

Reply:
> We should probably change the name of this function to distinguish it from the class Scattering2D. It's easy to confuse the two right now.

So, I proposed to use the same convention as pytorch. As an example: https://pytorch.org/docs/stable/_modules/torch/nn/modules/linear.html

> Another thing to think about is the directory structure. It may be better for the `scattering2d` function to be in a `core` submodule (or something like that). Right now, it's a little curious that the main user interface resides in a submodule, while the internal function implementing the transform resides in the module itself.

I do agree. For this one, I'm not sure what's the best solution. I'm up to any solution... maybe we could call it functional like pytorch? But then we do copy pytorch and tf people might not be happy.. I don't know what to do..!

Reply:
> So, I proposed to use the same convention as pytorch.

Ok. Makes sense (although `L` and `l` are more easily distinguishable than `S` and `s`).